### DISCUSSION TITLE
multivariate forcasting - using  AddObservedValuesIndicator and Imutation

### DISCUSSION THREAD

**Discussion by ShirleyMgit** [awslabs/gluonts#2664] [Q&A]
I am trying to build a multivariate forecasting model to my dataset that contains Nans.
The transformation using AddObservedValuesIndicator and gluonts.transform.RollingMeanValueImputation fails.
(I could do it when I used only univariate series). 
1. Can we exploit these methods in the multivariate case? Do you have any example? I did not understand how to implement it
2. General missing values question: if I do not want to impute the series and I want the network to ignore the missing value, should I introduce "observed values" without imputation? in that case the network will not predict these points in training and introduce them as zero in the network input? I am asking it, as in the univariate case it worked better with imputation, which is not intuitive to me and I am wondering how missing values are treated.
thanks

**Comment by ShirleyMgit** [comment]
1) I have extended it to multivariate myself.
2) from the code it seems that it is not implemented

**Reply by mbohlkeschneider** [reply]
Hi @ShirleyMgit,

1. Nice! would you share the PR? I would be curious how you did the multivariate imputation (which is probably not trivial to do).
2. You are correct, dealing with missing values is not implemented in `DeepVAR`
